_config_authoring.yml

```diff
@@ -2,36 +2,38 @@ name: Docker Documentation
 markdown: kramdown
 kramdown:
   input: GFM
+  gfm_quirks: [paragraph_end, no_auto_typographic]
   html_to_native: true
   hard_wrap: false
   syntax_highlighter: rouge
   toc_levels: 2..3
+incremental: false
 permalink: pretty
 safe: false
 lsi: false
 url: https://docs.docker.com
-incremental: false
 
-# Component versions -- address like site.docker_ce_stable_version
+# Component versions -- address like site.docker_ce_version
 # You can't have - characters in these for non-YAML reasons
 
 # TO USE ME:
 # jekyll serve --incremental --config _config_authoring.yml
 
-docker_ce_stable_version: "17.09"
-latest_stable_docker_engine_api_version: "1.32"
-docker_ce_edge_version: "17.09"
-docker_ee_version: "17.06"
-compose_version: "1.16.1"
-machine_version: "0.12.2"
+latest_engine_api_version: "1.39"
+docker_ce_version: "18.09"
```

Comment:
This is gonna conflict with https://github.com/docker/docker.github.io/pull/8010, but shouldn't be hard to fix once that's merged

Reply:
#8010 has been merged.